modules/client-configuration/pages/clients-slemicro.adoc

```diff
@@ -136,14 +153,14 @@ To add the client tools, add these channels from the command prompt:
 | OS Version
 | Client Channel
 
-| {micro} 5.3
-| sle-micro-5.3-uyuni-client
+| {micro} 5.1
```

Comment:
I changed the order to match what we show above from SCC. 5.1 first, then 5,2 and finally 5.3